sequencer/src/lib.rs

```diff
@@ -488,15 +482,6 @@ pub async fn init_node<P: PersistenceOptions, V: Versions>(
         ))
     };
 
-    // Wait for the CDN network to be ready if we're not using the P2P network
```

Comment:
@rob-maron I removed this wait and think that should be fine because we always have the libp2p enabled. Let me know if you see an issue with this.